README.md

```diff
@@ -29,7 +29,7 @@ mkdir -p perf_test_ws/src
 cd perf_test_ws/src
 git clone git@github.com:ApexAI/performance_test
 cd ..
-ament build --parallel --build-tests --cmake-args -DCMAKE_BUILD_TYPE=Debug --
+ament build --parallel --build-tests --cmake-args -DCMAKE_BUILD_TYPE=Release
```

Comment:
`--parallel` is known to be problematic under certain conditions. 

If a user wants build speed, this user will use this option. Otherwise, it's a premature optimization.

Reply:
I am not a patient person. ;)